e2e/playwright/native-file-menu.spec.ts

```diff
@@ -1,2375 +1,539 @@
-                  if (!menu) {
-                    return false
-                  }
-                  menu.click()
-                  return true
-                })
-            )
-            .toBe(true)
-          // Check that the command bar is opened
-          await expect(cmdBar.cmdBarElement).toBeVisible()
-          // Check the placeholder project name exists
-          const actual = await cmdBar.cmdBarElement
-            .getByTestId('command-name')
-            .textContent()
-          const expected = 'Settings · app · theme'
-          expect(actual).toBe(expected)
-        })
-        test('Home.File.Preferences.Theme color', async ({
-          tronApp,
-          cmdBar,
-          page,
-        }) => {
-          if (!tronApp) fail()
-          // Run electron snippet to find the Menu!
-          await page.waitForTimeout(100) // wait for createModelingPageMenu() to run
-          await expect
-            .poll(
-              async () =>
-                await tronApp.electron.evaluate(async ({ app }) => {
-                  if (!app || !app.applicationMenu) {
-                    return false
-                  }
-                  const menu = app.applicationMenu.getMenuItemById(
-                    'File.Preferences.Theme color'
-                  )
-                  if (!menu) return false
-                  menu.click()
-                  return true
-                })
-            )
-            .toBe(true)
-          const settings = page.getByTestId('settings-dialog-panel')
-          await expect(settings).toBeVisible()
-          const defaultUnit = settings.locator('#themeColor')
-          await expect(defaultUnit).toBeVisible()
-        })
-        test('Home.File.Preferences.Sign out', async ({
-          tronApp,
-          cmdBar,
-          page,
-        }) => {
-          if (!tronApp) fail()
-          // Run electron snippet to find the Menu!
-          await page.waitForTimeout(100) // wait for createModelingPageMenu() to run
-          await expect
-            .poll(
-              async () =>
-                await tronApp.electron.evaluate(async ({ app }) => {
-                  if (!app || !app.applicationMenu) return false
-                  const menu =
-                    app.applicationMenu.getMenuItemById('File.Sign out')
-                  if (!menu) {
-                    return false
-                  }
-                  // FIXME: Add back when you can actually sign out
-                  // menu.click()
-                  return true
-                })
-            )
-            .toBe(true)
-          // FIXME: When signing out during E2E the page is not bound correctly.
-          // It cannot find the button
-          // const signIn = page.getByTestId('sign-in-button')
-          // await expect(signIn).toBeVisible()
-        })
+test.describe('Native file menu', { tag: ['@electron'] }, () => {
+  test(
+    'Home page all in one test but uses steps',
```

Comment:
Up to you, but I don't think any other tests mention that they include steps in their name. This feels unnecessary to me.

```suggestion
    'Home page',
```